ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyflie.py

```diff
@@ -134,9 +331,38 @@ def cmdFullState(self, pos, vel, acc, yaw, omega):
         self.cmdFullStatePublisher.publish(self.cmdFullStateMsg)
 
     def cmdStop(self):
+        """Interrupts any high-level command to stop and cut motor power.
+
+        Intended for non-emergency scenarios, e.g. landing with the possibility
+        of taking off again later. Future low- or high-level commands will
+        restart the motors. Equivalent of :meth:`stop()` when in high-level mode.
+        """
         self.cmdStopPublisher.publish(std_msgs.msg.Empty())
 
     def cmdVel(self, roll, pitch, yawrate, thrust):
+        """Sends a streaming command of the "easy mode" manual control inputs.
+
+        The (absolute roll & pitch, yaw rate, thrust) inputs are typically
+        used for manual flying by beginners or causal pilots, as opposed to the
+        "acrobatic mode" inputs where roll and pitch rates are controlled
+        instead of absolute angles. This mode limits the possible maneuvers,
+        e.g. it is not possible to do a flip because the controller joystick
+        would need to rotate 360 degrees.
+
+        For more information on streaming setpoint commands, see the
+        :meth:`cmdFullState()` documentation.
+
+        TODO: should we change the name from cmdVel to something else?
+        IMO (japreiss), cmdVel implies controlling linear velocity.
+
+        Args:
+            roll (float): Roll angle. Radians. Positive values == roll right.
+            pitch (float): Pitch angle. Radians. Positive values == pitch
+                forward/down.
+            yawrate (float): Yaw angular velocity. Radians / second. Positive
+                values == turn counterclockwise.
```

Comment:
I think those are actually degrees and deg/s in CRTP. Is that converted somewhere?

Reply:
It's true for the manual control CRTP packet and for the firmware's internal setpoint struct, but not for the FullState CRTP packet. I wanted to give that packet "good" units and optimistically thought I would fix all the other packets soon :(